contributing file (#451)
metalbear-co/mirrord: CONTRIBUTING.md

## Contributing

> Our developer guide is [here](https://mirrord.dev/docs/developer/testing/). Before submitting features, please discuss them with us first. You can open an issue or a discussion for that matter.

Reviewer: ```suggestion
Our developer guide is [here](https://mirrord.dev/docs/developer/testing/). Before submitting pull request features, please discuss them with us first by opening an issue or a discussion.
```

> We welcome new/junior/starting developers and are willing to provide mentorship/guidance. Feel free to join to our [Discord channel](https://discord.gg/pSKEdmNZcK) for help and guidance.

Reviewer: ```suggestion
We welcome new/junior/starting developers. Feel free to join to our [Discord channel](https://discord.gg/pSKEdmNZcK) for help and guidance.
```
